lib/appsignal/transaction.rb

```diff
@@ -44,6 +47,40 @@ def create(namespace)
         end
       end
 
+      # @api private
```

Comment:
👍 Let's keep it private for now, so we can also consider using it in the context of https://github.com/appsignal/integration-guide/issues/77 before making it public.